include/deal.II/fe/fe_tools.templates.h

```diff
@@ -2622,11 +2622,11 @@ namespace FETools
                     // find sub-quadrature
                     position = name.find('(');
                     const std::string subquadrature_name(name, 0, position);
-                    AssertThrow(subquadrature_name.compare("QTrapez") == 0,
```

Comment:
I think this needs more work because the name could be QTrapez or QTrapezoid here, right? (assuming somebody has an FE name in a .prm file.

Reply:
Also: is the string "QTrapez" generated somewhere in the code? I could not find it.

Reply:
Yes, it requires a couple more lines. Fixed.